### ISSUE TITLE
Error while installing Azure DevOps integration

### ISSUE THREAD

**Issue by InterstellarStella** [getsentry/sentry#81229]
### Environment

SaaS (https://sentry.io/)

### Steps to Reproduce

1. Have a publicly available Azure DevOps Organization
2. Follow the documented steps to install the Azure DevOps integration, making sure to have all the necessary permissions.

### Expected Result

The integration is installed successfully.

### Actual Result

Getting the error "An error occurred while validating your request". Notice the `access_denied` part int he URL.
![Image](https://github.com/user-attachments/assets/5bbd7d83-496d-4cc3-84dc-be529a195247)

More details in the shadow ticket.
The corresponding transaction in Sentry's Sentry is not showing any errors.

### Product Area

Settings - Integrations

**Comment by sentaur-athena** [comment]
@GabeVillalobos is investigating this. We will post updates soon.

**Comment by GabeVillalobos** [comment]
I found logs surrounding the install attempts on the 14th, but this occurred before we fixed some of the logging around our integration installs. @InterstellarStella could you please ask the customer to retry installing Azure DevOps?

**Comment by roofiq** [comment]
Hello @GabeVillalobos , I am the customer that is facing those errors. We have just retried the Azure DevOps installation. Around 11:35 AM CEST. 
Thanks for checking it out :)

**Comment by roofiq** [comment]
Anyny update with this? @GabeVillalobos ?

**Comment by iamrajjoshi** [comment]
Hello @roofiq thanks for reaching out, I would just like to confirm if the user installing the integration has "Organization Administrator Permissions" for the DevOps instance.  Access denied seems to point to a permission issue when installing the integration. We aren't relieving any additional context from Azure DevOps for this.

**Comment by roofiq** [comment]
@iamrajjoshi yes, I can confirm that requesting account has Organization Owner in Azure DevOps

**Comment by sentaur-athena** [comment]
@roofiq we're working with @InterstellarStella to figure out the next steps. Unfortunately we can't debug further on our side since this is an error thrown by Azure about permissions.

**Comment by roofiq** [comment]
Hello guys, so I think I know whats up with this. 

I was trying to perform installation on the user that has application adminsitrator role in Azure and has access to single project, this error popped up:
![Image](https://github.com/user-attachments/assets/a4e0aaaf-e1ef-4970-a372-d1e5be7d53de)


I think that for a succesfull installation, you are asuming that user would have both roles enabled (on azure and azure devops). But it is not working like that in most of the organization. For example we have spearate account with admin access to Azure and other one to Azure DevOps. So I think this is where issue come from.

Would it be possible to catch up for a short demo when we will show you how we are trying to pass through installation?

Regards.
Rafal

**Comment by sentaur-athena** [comment]
@InterstellarStella if you're scheduling a meeting, please make sure to include @iamrajjoshi

**Comment by InterstellarStella** [comment]
After the call me and @iamrajjoshi had with the customer, investigation will continue through this GH issue.

**Comment by roofiq** [comment]
@InterstellarStella hello, do we have any update on this one?

**Comment by InterstellarStella** [comment]
@iamrajjoshi Is there an update we can provide for @roofiq ?

**Comment by sentaur-athena** [comment]
@InterstellarStella let's sync offline.

**Comment by sentaur-athena** [comment]
@roofiq unfortunately we have to delay this fix till April since @iamrajjoshi is pulled into a high priority project that will take all his time. This is the first item on his list after that project though.

**Comment by iamrajjoshi** [comment]
@roofiq  We recently made some changes around consent, could you please try again and let me know if the issue persists?
